docs/ABOUT.md

```diff
@@ -1,14 +1,5 @@
-R is a free, open source, programming language and software environment for statistical computing and graphics developed at Bell Laboratories as a GNU project. It is designed as a true computer language, and is written in C, Fortran and R.
+R is an open source programming language for data analysis and statistical computing. It was created by Ross Ihaka and Robert Gentleman at the University of Auckland, and is an implementation of the S programming language combined with lexical scoping semantics inspired by Scheme. The language is interpreted, dynamically typed, and has a strong foundation in functional programming. This makes it well suited to many of the challenges of data analysis.
```

Comment:
Thank you so much for this review and feedback! I agree that this opening is a little too heavy. 

I have simplified this a lot and highlighted the inspiration behind the creation of the language as suggested. Also emboldened the **R**'s in the creators names (since they have noted that this is partly where the name of the language came from).

The rest remains the same apart from the addition of a quote from one of the creators in the 3rd paragraph, and the inclusion of a link to the project's home page at the end.